SonarQube.MSBuild.Tasks/GetAnalyzerSettings.cs

```diff
@@ -57,25 +63,38 @@ public override bool Execute()
 
             AnalysisConfig config = TaskUtilities.TryGetConfig(this.AnalysisConfigDir, new MSBuildLoggerAdapter(this.Log));
             
-            if (config != null)
+            if (config != null && Language != null)
```

Comment:
why is this check required? wouldn't adding `[Required]` on the `Language` attribute help?